examples/BUILD.bazel

```diff
@@ -67,4 +137,22 @@ java_binary(
     runtime_deps = [
         ":opencensus_examples",
     ],
+)
+
+java_binary(
+    name = "HelloWorldClient",
+    testonly = 1,
```

Comment:
Why are the new build targets `testonly`?

Reply:
Done. I copied this from gRPC java build: https://github.com/grpc/grpc-java/blob/master/examples/BUILD.bazel#L85. Not sure why they specify this as test-only though.